keyboards/ergodox_ez/keymaps/default/keymap.c

```diff
@@ -115,94 +116,95 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
                                       KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, KC_WBAK
 ),
 };
+// clang-format on
 
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
-  if (record->event.pressed) {
-    switch (keycode) {
-      case VRSN:
-        SEND_STRING (QMK_KEYBOARD "/" QMK_KEYMAP " @ " QMK_VERSION);
-        return false;
-      #ifdef RGBLIGHT_ENABLE
-      case RGB_SLD:
-        rgblight_mode(1);
-        return false;
-      #endif
+    if (record->event.pressed) {
+        switch (keycode) {
+            case VRSN:
+                SEND_STRING(QMK_KEYBOARD "/" QMK_KEYMAP " @ " QMK_VERSION);
+                return false;
+#ifdef RGBLIGHT_ENABLE
+            case RGB_SLD:
+                rgblight_mode(1);
+                return false;
+#endif
```

Comment:
Couldn't this be replaced with `RGB_M_P`?

Reply:
Yup, and done